### PR TITLE
profile links to social are external

### PR DESCRIPTION
**Pull request by RomaneGreen**
profile links to are now external

### CODE REVIEW

**Comment by crutchcorn** [comment]
Hey there! Thanks for the contribution! 

If you could revert the package lock file we can get this merged in! 😁

**Comment by crutchcorn** [comment]
Also, we're wanting to make pull requests against the `integration` branch (I really aught to put this in the README, sorry for not doing so)

File: src/components/pic-title-header/pic-title-header.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const SocialBtn = ({icon, text, url, name}) => {
     if (name.endsWith('s')) return `${name}'`;
     return `${name}'s`;
   }, [name])
-  return <OutboundLink className='unlink baseBtn lowercase prependIcon' href={url}>
+  return <OutboundLink className='unlink baseBtn lowercase prependIcon' target="_blank" href={url} >
```
Comment (evelynhathaway) [inline src/components/pic-title-header/pic-title-header.js]:
```suggestion
  return <OutboundLink className='unlink baseBtn lowercase prependIcon' target="_blank" rel="noopener" href={url} >
```

[Read more on `noopener` for external links](https://developers.google.com/web/tools/lighthouse/audits/noopener)

**Comment by crutchcorn** [comment]
Closing in favor of #59 (the other PR opened from @RomaneGreen )

**Pull request closed without merging**